srcpkgs/TheForceEngine/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'TheForceEngine'
+pkgname=TheForceEngine
+version=1.09.530
+revision=1
+build_style=cmake
+hostmakedepends="pkg-config"
+makedepends="SDL2-devel devil-devel rtmidi-devel jack-devel glew-devel MesaLib-devel"
+depends="zenity"
+short_desc="Modern \"Jedi Engine\" replacement supporting Dark Forces and mods"
+maintainer="Kenneth Dodrill <hello@kennydodrill.com>"
+license="GPL-2.0-only"
+homepage="https://theforceengine.github.io"
+distfiles="https://github.com/luciusDXL/TheForceEngine/archive/v${version}.tar.gz"
+checksum=6b4649fe219ec0678888e7b2076017cda545f97218d5d05f29cb7604bf45097e
+
+case "$XBPS_TARGET_MACHINE" in
+	i686*) broken="Minimum requirements state 64-bit for GPU renderer" ;;
+	ppc*) broken="Minimum requirements state 64-bit for GPU renderer" ;;
+	arm*) broken="Minimum requirements state 64-bit for GPU renderer" ;;
+	*-musl) broken="FAILED: CMakeFiles/tfe.dir/TheForceEngine/TFE_FileSystem/fileutil-posix.cpp.o" ;;
+esac
```

Comment:
```suggestion
if [ "$XBPS_TARGET_WORDSIZE" == "32" ]; then
	broken="Minimum requirements state 64-bit for GPU renderer" ;;
fi
```

Checking `XBPS_TARGET_WORDSIZE` instead of listing all 32bit architectures is more sensible.

Reply:
Much simpler, thank you!